scripts/clog.inputs

```diff
@@ -0,0 +1,124 @@
+../src/platform/selfsign_openssl.c
```

Comment:
I'm not a fan of requiring this file. It's yet another file we have to maintain.